pyne/ensdf.py

```diff
@@ -69,20 +69,72 @@ def _to_id(nuc):
     return nucid
 
 
+# Energy to half-life conversion:  T1/2= ln(2) × (h/2 pi) / energy
+# See http://www.nndc.bnl.gov/nudat2/help/glossary.jsp#halflife
+# NIST CODATA https://physics.nist.gov/cgi-bin/cuu/Value?hbar
+#    h-bar = 1.054 571 800(13) x 1e-34 J
+#    1 J = 6.241 509 126(38) x 1e18 eV
+HBAR_LN2 = 4.5623775832376968e-16  # h-bar ln(2) in eV s
+energy_conv_dict = {'ev': HBAR_LN2,
+                    'kev': 1e-3 * HBAR_LN2,
+                    'mev': 1e-6 * HBAR_LN2,
+                    }
+
+
+def _halflife_to_seconds(value, err, units):
+    """Converts a halflife with err and units to seconds.
+
+    Parameters
+    ----------
+    value: number
+        Time or energy, depending on units.
+    err : number or (number, number)
+        Uncertainty, or (plus, minus) uncertainty in [units].
+    units : str
+        Units flag, eg 'min', 'ms', 'days', or even 'MeV'.
+
+    Returns
+    -------
+    sec_time : float
+        Time value in [sec].
+    sec_err : None or float or (float, float) in [sec].
+        Time uncertainty in [sec], or (plus, minus) if asymmetric uncertainty.
+    """
+    if err is None:
+        plus, minus = 0, 0
+    elif isinstance(err, float):
```

Comment:
This might fail in unexpected ways.  For example, `isinstance(np.float32(10.0), float)` is False.  It seems like it might be better to test if it is a Sequence.  

```python
from collections import Sequence

...

elif isinstance(err, Sequence):
   plus, minus = err
else:
    plus = minus = err
```

Reply:
I disagree, since this is an internal function called by the parser which converts error strings to None, float, or (float, float).   Yes, if called directly by user code it is quite possible that an int or a numpy scalar or even a mpmath 500 digit constant, but it won't be in this code.  Slowly introducing PEP484 and mypy typing is a better solution since you only pay for it at build-time, not at run-time.

That said, I will put in a new commit with a np.isscalar() test.